### PR TITLE
initial structure set up

### PR DESCRIPTION
**Pull request by ychen-usgs**
Added 1_fetch and 2_process; moved my_happy_script.R

### CODE REVIEW

File: 2_process/src/diag_func.R [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+diag_func <- function (data,out_dir) {
+  
+  library(dplyr)
+  library(stringr)
+  library(whisker)
```
Comment (lindsayplatt) [inline 2_process/src/diag_func.R]: It's a good idea to load libraries _outside_ of functions (so I would suggest just moving this up above `diag_func()`

File: 2_process/src/fetch_func.R [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  library(readr)
+  library(stringr)
+  library(sbtools)
+  
+  # Get the data from ScienceBase
+  mendota_file <- file.path(download_dir, 'model_RMSEs.csv')
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
+  
+  # Prepare the data for plotting
+  eval_data <- readr::read_csv(mendota_file, col_types = 'iccd') %>%
```
Comment (lindsayplatt) [inline 2_process/src/fetch_func.R]: We want to separate this function into a `fetch` step (which should go into a `1_fetch/src/` folder and a process step, which can live in `2_process/src`

File: 2_process/src/run_script.R [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# defining variables
+download_dir <- '1_fetch/out/'
+out_dir <- '2_process/out/'
+eval_data <- fetch_data(download_dir)
+
+# rendering outputs
+plot_func(eval_data,out_dir)
+render_func(eval_data,out_dir)
+diag_func(eval_data,out_dir)
```
Comment (lindsayplatt) [inline 2_process/src/run_script.R]: Nice job outlining how I would run this!

**Review by lindsayplatt — CHANGES_REQUESTED** [review]
Just two small comments!

**Comment by ychen-usgs** [comment]
How I would run my functions:

source("1_fetch/src/fetch_func.R")
source("2_process/src/clean_func.R")
source("3_visualize/src/ploting_func.R")
source("2_process/src/render_func.R")
source("2_process/src/diag_func.R")

# defining variables
download_dir <- '1_fetch/out/'
out_dir <- '2_process/out/'
vis_out <- '3_visualize/out/
data_input <- fetch_data(download_dir)
eval_data <- clean_func(data_input)

# rendering outputs
plot_func(eval_data,vis_out)
render_func(eval_data,out_dir)
diag_func(eval_data,out_dir)

**Review by lindsayplatt — APPROVED** [review]
Looks great!

**Pull request merged**